Spokestack/TextToSpeech.swift

```diff
@@ -172,3 +184,15 @@ import AVFoundation
         }
     }
 }
+
+fileprivate struct SynthesizeResponseURL: Codable {
```

Comment:
Is this one ever used?

Reply:
yes, by the struct immediately below it

Reply:
Weird; I saw that relationship in the second and third, but my brain didn't register this one for some reason.